model-utils/src/test/java/io/github/ygojson/model/utils/schema/acceptance/JsonSchemaGeneratorAcceptanceTest.java

```diff
@@ -0,0 +1,50 @@
+package io.github.ygojson.model.utils.schema.acceptance;
+
+import java.io.IOException;
+
+import org.approvaltests.Approvals;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import io.github.ygojson.model.utils.schema.JsonSchemaGenerator;
+
+class JsonSchemaGeneratorAcceptanceTest {
```

Comment:
Should add a test to check if nested jackson description annotations behave and fix whatever is required if not as expected.

Reply:
Probably at the unit test level

Reply:
Opened #27  for that use-case as it is not working as expected.